DeviceWhoAmI.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$id": "https://harp-tech.org/2023-07/whoami",
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "definitions": {
+        "device": {
+            "properties": {
+                "authors": {
+                    "type": "string"
+                },
+                "name": {
+                    "type": "string",
+                    "pattern": "^[a-zA-Z][a-zA-Z0-9_]*$"
+                },
+                "copyright": {
+                    "type": "string"
+                },
+                "projectUrl": {
+                    "type": "string"
+                },
+                "sourceUrl": {
```

Comment:
Cannot see my original comment anymore so maybe I made a mistake, but this should be `repositoryUrl` instead of `sourceUrl`